daemon/oci_windows.go

```diff
@@ -336,12 +338,16 @@ func (daemon *Daemon) createSpecWindowsFields(c *container.Container, s *specs.S
 // Sets the Linux-specific fields of the OCI spec
 // TODO: @jhowardmsft LCOW Support. We need to do a lot more pulling in what can
 // be pulled in from oci_linux.go.
-func (daemon *Daemon) createSpecLinuxFields(c *container.Container, s *specs.Spec) {
+func (daemon *Daemon) createSpecLinuxFields(c *container.Container, s *specs.Spec) error {
 	if len(s.Process.Cwd) == 0 {
 		s.Process.Cwd = `/`
 	}
 	s.Root.Path = "rootfs"
 	s.Root.Readonly = c.HostConfig.ReadonlyRootfs
+	if err := setCapabilities(s, c); err != nil {
+		return fmt.Errorf("linux spec capabilities: %v", err)
```

Comment:
Perhaps "failed to set ....." ?

Also may want to use `errors.Wrap(err, "failed to set ...")` to preserve the original error

Reply:
I copied these errors from the Linux use of the functions. Do you want me to update both places or leave it as is?

(Maybe better to avoid duplication altogether but doing that in this change seemed to be more trouble than its worth right now.)

Reply:
Ah! Didn't check be Linux equivalent, but now recall I saw those at some point and wanted to update them.

Let's keep them as-is for now to be consistent, and keep it for a separate PR to improve if we want to